deliveries/prosopo-milestone-2.md

```diff
@@ -0,0 +1,105 @@
+# Milestone Delivery :mailbox:
+
+**
+The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform)
+has been filled out correctly for this milestone and the delivery is according to the
+official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md)
+.**
```

Comment:
```suggestion
**
The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**
```

Reply:
this will fix the formatting

Reply:
> Hey @forgetso Happy to hear from you as well! I followed your advice by running the following commands on the root directory, however, it fails at `npm run setup`:
> 
> ```shell
> $ git checkout milestone2
> $ git log
> commit 3a6537138de2347b826e301620997cf30b6a5a8c (HEAD -> milestone2, origin/milestone2)
> Author: Chris Taylor <forgetso86@gmail.com>
> Date:   Tue Aug 2 11:35:15 2022 +0100
>     Remove copy command from readme
> $ git submodule update
> Submodule path 'client-example': checked out '7268766bb1b3fcfe3cbdf18b1ad6ba8d62994586'
> Submodule path 'dapp-example': checked out '6ce098640abc6100b0e8d9d08184da7a418c9bc6'
> Submodule path 'demos/demo-nft-marketplace': checked out '7978433306fec284cf024452eb3d5134953dac0f'
> Submodule path 'packages/provider': checked out 'f6a454fb194fe877543c3549dbf8c99ed08e447b'
> $ npm i
> up to date, audited 2882 packages in 3s
> $ npm run setup && npm run start # setup provider
> 
> > integration@0.0.0 setup
> > npm -w @prosopo/provider run setup
> 
> 
> > @prosopo/provider@0.0.0 setup
> > node ./dist/cli/setup.js
> 
> node:internal/modules/cjs/loader:936
>   throw err;
>   ^
> 
> Error: Cannot find module '/home/xxx/repos/integration/packages/provider/dist/cli/setup.js'
>     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
>     at Function.Module._load (node:internal/modules/cjs/loader:778:27)
>     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
>     at node:internal/main/run_main_module:17:47 {
>   code: 'MODULE_NOT_FOUND',
>   requireStack: []
> }
> npm ERR! Lifecycle script `setup` failed with error: 
> npm ERR! Error: command failed 
> npm ERR!   in workspace: @prosopo/provider@0.0.0 
> npm ERR!   at location: /home/xxx/repos/integration/packages/provider
> ```
> 
> Could you please double-check?

Sorry, I see I have not pasted the entire `git submodule` command! The setup script is missing because the submodules are not pulled. Please try this command:

`git submodule update --init --recursive --force --checkout`

*Not* this command:

`git submodule update`

I'll update the docs.

Reply:
https://github.com/prosopo-io/integration/tree/milestone2#install-node-modules